src/main/java/wordle/ui/ResultView.java

```diff
@@ -0,0 +1,38 @@
+package wordle.ui;
+
+import java.util.List;
+import wordle.domain.BingoHistory;
+
+public class ResultView {
+    private static final int MAX_TRY_COUNT = 6;
+
+    public void printGameStart() {
+        System.out.println("WORDLE을 6번 만에 맞춰 보세요.");
+        System.out.println("시도의 결과는 타일의 색 변화로 나타납니다.");
```

Comment:
```suggestion
        System.out.println("WORDLE을 6번 만에 맞춰 보세요.");
        System.out.println("시도의 결과는 타일의 색 변화로 나타납니다.");
```
```suggestion
        private static final GAME_START_MESSAGE = "WORDLE을 6번 만에 맞춰 보세요.";
        private static final GAME_RULE_MESSAGE = "시도의 결과는 타일의 색 변화로 나타납니다.";
        
        System.out.println(GAME_START_MESSAGE);
        System.out.println(GAME_RULE_MESSAGE);
```

출력 메시지를 상수화해서 처리하는 것은 어떨까요?